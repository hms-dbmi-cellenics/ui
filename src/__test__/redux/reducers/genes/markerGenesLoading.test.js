import markerGenesLoadingReducer from 'redux/reducers/genes/markerGenesLoading';

describe('markerGenesLoading', () => {
  it('returns correct state', () => {
    const initialState = {
      markers: {},
    };

    const newState = markerGenesLoadingReducer(initialState, {});

    expect(newState).toMatchSnapshot();
  });
});
