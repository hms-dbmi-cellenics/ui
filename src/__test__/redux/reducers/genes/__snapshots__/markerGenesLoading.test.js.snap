// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`markerGenesLoading returns correct state 1`] = `
{
  "expression": {
    "downsampled": {
      "ETag": "new-etag",
    },
  },
  "markers": {
    "error": false,
    "loading": true,
  },
}
`;
